taprpc/taprootassets.proto

```diff
@@ -1541,6 +1541,24 @@ message ListBurnsResponse {
     repeated AssetBurn burns = 1;
 }
 
+// A transaction outpoint annotated with TAP-level asset metadata. It uniquely
+// identifies an asset anchored at a specific outpoint.
+message AssetOutPoint {
```

Comment:
nit: since we're adding this as a new type, this could go into the `tapcommon.proto` instead.